lib/utils/utils.go

```diff
@@ -452,6 +452,12 @@ func GetFreeTCPPorts(n int, offset ...int) (PortList, error) {
 	return PortList{ports: list}, nil
 }
 
+// HostUUIDExistsLocaly checks if dataDir/host_uuid file exists in local storage.
+func HostUUIDExistsLocaly(dataDir string) bool {
```

Comment:
I'd just use `ReadHostUUID` and discard the string value, either instead of this function or instead of `ReadPath` here.